exercises/meetup/description.md

```diff
@@ -1,14 +1,20 @@
-Typically meetups happen on the same day of the week.
+Typically meetups happen on the same day of the week.  In this exercise, you will take
+a description of a meetup date, and return the actual meetup date.
 
-Examples are
+Examples of general descriptions are:
 
-- the first Monday
-- the third Tuesday
-- the Wednesteenth
-- the last Thursday
+- the first Monday of January 2017
+- the third Tuesday of January 2017
+- the Wednesteenth of January 2017
+- the last Thursday of January 2017
 
 Note that "Monteenth", "Tuesteenth", etc are all made up words. There
-was a meetup whose members realised that there are exactly 7 days that
+was a meetup whose members realized that there are exactly 7 numbered days in a month that
 end in '-teenth'. Therefore, one is guaranteed that each day of the week
 (Monday, Tuesday, ...) will have exactly one date that is named with '-teenth'
 in every month.
+
+Given examples of a meetup dates, each containing a month, day, year, and descriptor 
+("first, second, teenth, etc"), calculate the date of the actual meetup.
```

Comment:
I might remove the quotes entirely, or quote each item separately (`"first", "second"...`) but not etc., since we wouldn't see `etc` as a descriptor